docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: '3'
+
+services:
+  base: &base
+    platform: linux/amd64
+    build: ./api/
+    image: mlenabler/base
+    command: sh -c "echo I am base!"
+    container_name: ml-enabler-base
+    restart: always
+    links:
+      - postgis
+    depends_on:
+      - postgis
+    environment:
+      - PORT=2001
+      - POSTGRES=postgres://docker:docker@postgis:5432/gis
+      - MAPBOX_TOKEN
+    volumes:
+      - /home/mle/api/node_modules
+
+  api:
+    <<: *base
+    container_name: ml-enabler-api
+    command: sh -c "sleep 5; npm run dev"
```

Comment:
do we know why we have these `sleep`s in here? Just curious.

Reply:
As all these containers are created in parallel, this will allow time for PostGIS DB to be created & populated.